src/core/components/Header/components/Category/Category.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { FC, useState } from "react";
+import arrowDownThin from "@/assets/images/arrow_down_thin.svg";
+import "./Category.scss";
+
+const brands = ["Samsung", "Sony", "Apple"];
+
+interface CategoryProps {
+  title: string;
+}
+
+const Category: FC<CategoryProps> = ({ title }) => {
+  const [showMenu, setShowMenu] = useState<boolean>(false);
+
+  const handleHideMenu = () => {
+    setShowMenu(false);
+  };
+
+  const handleShowMenu = () => {
+    setShowMenu(true);
+  };
+
+  return (
+    <div className="category" onMouseLeave={handleHideMenu}>
+      <button
+        className="category__btn"
+        type="button"
+        onMouseEnter={handleShowMenu}>
+        {title}
+        <img
+          className={`${showMenu && "reverse__icon"}`}
+          src={arrowDownThin}
+          alt="DownArrow"
+        />
+      </button>
```

Comment:
Should it be a button ?

Reply:
Replace button with div, but I did not understand - why I can`t use here button.

Reply:
Because button it's something clickable, something that trigger some action
This one works on hover

If the dropdown only appears when the button is clicked, I agree it could be a button